etc/config-template.toml

```diff
@@ -178,6 +178,10 @@
 ## The default value is 2 if the number of CPU cores is less than 16, otherwise 3.
 # background-thread-count = 2
 
+## If handle time is larger than the threshold, it will print slow log in end point.
+## The default value is 1s.
+# end_point_slow_log_threshold = "1s"
```

Comment:
@breeswish I suggest using "endpoint". It's a specific name to TiKV.

Reply:
In configuration item it is end-point:

```
# end-point-recursion-limit = 1000
```

You remind me that this configuration item should be hyphen-connected, instead of underscore_connected.